homeassistant/components/zwave/__init__.py

```diff
@@ -793,8 +795,11 @@ def value_changed(self):
         self.update_properties()
         # If value changed after device was created but before setup_platform
         # was called - skip updating state.
-        if self.hass:
-            self.schedule_update_ha_state()
+        if self.hass and not self._scheduled_update:
+            self._schedule_update()
```

Comment:
Changed to hass.add_job as `async_add_job` is only supposed to be called from a loop